### DISCUSSION TITLE
Make custom author fields in bulk editing consistent if the series have it

### DISCUSSION THREAD

**Discussion by noaione** [gotson/komga#1562] [Ideas]
### Describe your suggested feature

I don't know how to title it properly, but my request is to make the custom author fields consistent when bulk editing a books if the series have any custom one.

For example:
1. Series A have an extra "Character Design" fields and applied to book 1-5
2. Series A got updated with new book (6-10)
3. If we try to bulk edit 6-10, there will be no "Character Design" field, but if we do 1-10, 5-10, etc, there will be "Character Design" field.

My hope is that the custom field will be consistent even if that new books does not have the custom fields yet, it would make tagging much faster.

### Other details

The below custom fields I mentioned, I only need the field itself not having it filled with anything.

![https://p.ihateani.me/nevqgtax.png](https://p.ihateani.me/nevqgtax.png)

I think bulk metadata editing also can be done from the Recommended/home pages, so maybe check if we're editing the same series too if the custom fields should be visible or not?

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
When you edit books, the author roles shown are the ones from all the books.

What you are asking is to retrieve all roles available for the books' parent series. But as you mentioned you can edit books from different series, in which case, what would happen ? Show the roles from all those series ? Or the current behaviour?

**Comment by noaione** [comment]
Keep the current behaviour would be better
